walkthroughs/eks/o11y-jaeger.md

```diff
@@ -25,7 +28,8 @@ helm upgrade -i appmesh-controller eks/appmesh-controller \
     --set tracing.enabled=true \
     --set tracing.provider=jaeger \
     --set tracing.address=<JAEGER_ENDPOINT_ADDR / JAEGER_SERVICE_NAME> \
-    --set tracing.port=<JAEGER_ENDPOINT_PORT>
+    --set tracing.port=<JAEGER_ENDPOINT_PORT> \
+    --set serviceAccount.create=false
```

Comment:
is serviceAccount.create is needed ?

Reply:
Yeah, otherwise error was thrown, complaining something about serviceAccount name doesn't exists. I don't have the error message handy, but can reproduce if needed